package.json

```diff
@@ -14,9 +14,13 @@
     "gulp-less": "^3.0.2",
     "gulp-livereload": "^3.8.0",
     "gulp-minify-css": "^1.0.0",
+    "gulp-newer": "^0.5.0",
     "gulp-rename": "^1.2.2",
     "gulp-sourcemaps": "^1.5.1",
     "less": "~2",
-    "source-map": "^0.4.2"
+    "requirejs": "^2.1.17",
+    "source-map": "^0.4.2",
+    "through": "^2.3.7",
```

Comment:
You may already know this, but I found recently that you can write this as `2.x` - if you prefer, not necessarily suggesting you do that here. https://github.com/npm/node-semver